### PR TITLE
languagetool 3.8

### PR DESCRIPTION
**Pull request by techaddicted**
Created with `brew bump-formula-pr`.

### CODE REVIEW

**Comment by techaddicted** [comment]
Looks like some kind of git clone error in the CI:

> 08:58:40 Caused by: hudson.plugins.git.GitException: Command "git fetch --tags --progress https://github.com/Homebrew/homebrew-core/ +refs/pull/*:refs/remotes/origin/pr/*" returned status code 128:
08:58:40 stdout: 
08:58:40 stderr: error: RPC failed; curl 18 transfer closed with outstanding read data remaining
08:58:40 fatal: The remote end hung up unexpectedly

File: Formula/languagetool.rb [hunk 1]
```diff
@@ -1,8 +1,8 @@
 class Languagetool < Formula
   desc "Style and grammar checker"
   homepage "https://www.languagetool.org/"
-  url "https://languagetool.org/download/LanguageTool-3.7.zip"
-  sha256 "a97763a32e15b3f7d16e235b0137462d9699548d9d5a88b2f7764c21c04f2412"
+  url "https://languagetool.org/download/LanguageTool-3.8.zip"
+  sha256 "e910a4ea0f0000409ba91220cbf56ae0f23c814efab5d2845f8059cbb918453c"
 
   bottle :unneeded
```
Comment (fxcoudert) [inline Formula/languagetool.rb]: Please add a `depends_on :java => "1.8"` line after the `bottle :unneeded`

**Review by fxcoudert — CHANGES_REQUESTED** [review]

File: Formula/languagetool.rb (hunk 1 above)
Comment (techaddicted) [inline Formula/languagetool.rb]: Thanks for the information 👍

**Review by fxcoudert — APPROVED** [review]

**Comment by fxcoudert** [comment]
Thanks! (Next time, please squash into a single commit, too)

**Pull request closed without merging**